janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1785,7 +1786,7 @@ private void testNestedWrites(String initialValue, String updatedValue, TestInfo
 
         // Write schema and one vertex
         final PropertyKey prop = makeKey(propName, String.class);
-        createExternalVertexIndex(prop, INDEX);
+        mgmt.buildIndex("mixed", Vertex.class).addKey(prop, Mapping.STRING.asParameter()).buildMixedIndex(INDEX);
```

Comment:
What's the logical difference between this line and the original one?

Reply:
The original one builds a mixed index backed by TEXT, which supports full-text search but not equality check. Therefore, original line 1816
```java
assertEmpty(graph.query().has(propName, updatedValue).vertices());
```
does a full scan without using mixed index.